addons/point_of_sale/__manifest__.py

```diff
@@ -26,6 +26,7 @@
         'views/pos_category_view.xml',
         'views/product_view.xml',
         'views/account_journal_view.xml',
+        'views/pos_payment_method_views.xml',
```

Comment:
put two lines together at the end

Reply:
I can't. There is an action defined in 'views/pos_payment_method_views.xml' that is needed in 'views/pos_config_view.xml'. Instead, can I put 'views/pos_payment_views.xml' beside 'views/pos_payment_method_views.xml'?

Reply:
ok, no problem